app/src/main/java/io/intrepid/contest/screens/contestcreation/NewContestActivity.java

```diff
@@ -28,12 +29,11 @@
 import timber.log.Timber;
 
 public class NewContestActivity extends BaseMvpActivity<NewContestPresenter> implements NewContestMvpContract.View, EditContestContract {
-    @BindView(R.id.toolbar)
-    Toolbar toolbar;
     @BindView(R.id.fragment_container)
     ViewPager viewPager;
     private SlidingTabAdapter tabAdapter;
     private ActionBar actionBar;
+    private Contest.Builder contest;
```

Comment:
Actually looks like we might not be using this variable yet. We can remove it and add it back when we need it.